internal/controller/standalone_pgadmin/pod.go

```diff
@@ -277,7 +267,11 @@ func startupScript(pgadmin *v1beta1.PGAdmin) []string {
 
 	// startCommands (v8 image includes Gunicorn)
 	var startCommandV7 = "pgadmin4 &"
-	var startCommandV8 = "gunicorn -c /etc/pgadmin/gunicorn_config.py --chdir $PGADMIN_DIR pgAdmin4:app &"
+	// For Gunicorn, watch the logging config and reload if changes are detected.
+	var startCommandV8 = "gunicorn -c /etc/pgadmin/gunicorn_config.py" +
+		" --reload-extra-file " + configMountPath + `/` + gunicornLogConfFilePath +
+		" --log-config-json " + configMountPath + `/` + gunicornLogConfFilePath +
```

Comment:
👍🏻 Nice! We can do a lot with config files.

❓ My read of [`reload_extra_files`](https://docs.gunicorn.org/en/latest/settings.html#reload-extra-files) is that it requires `reload` to be enabled. Do we use that?

❓ I see `reload_extra_files` and `logconfig_json` mentioned in the config file settings. Is it possible to set these through `settings` in `generateGunicornConfig()`? 🤔 If so, we might end up with all JSON and no change to the entrypoint?

Reply:
See https://github.com/benoitc/gunicorn/issues/3353 for an issue I ran into.

Not in that GH issue, but as a precursor to it: all the `reload` stuff is expressly for development. So I was thinking of adding a SIGHUP signal into our mtime checking loop, only SIGHUP doesn't work -- but SIGKILL might if we want to do that every time a cluster changes or the config changes. (The mtime for all those files change together since they are in the same configmap.)